tests/integ/test_tuner_multi_algo.py

```diff
@@ -190,6 +190,7 @@ def _deploy_and_predict(sagemaker_session, tuner, data_set, cpu_instance_type):
             )
         )
         predictor = tuner.deploy(1, cpu_instance_type, endpoint_name=best_training_job)
+        predictor.content_type = "application/json"
```

Comment:
In my next PR, I'm going to add serializer and deserializer parameters to `deploy`. In that PR, I'll remove `predictor.content_type = ...`.